### PR TITLE
Update `yapf` style

### PR DESCRIPTION
**Pull request by sadym-chromium**
Update `yapf` style to increase readability of the 
```
"prop_1":
"value",
"prop_2":
"value",
```

to 

```
"prop_1": "value",
"prop_2": "value",
```

Related issue: https://github.com/google/yapf/issues/392

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ repos:
     rev: v0.32.0
     hooks:
       - id: yapf
-        args: ['--in-place', '--parallel']
+        args: ['--in-place', '--parallel', '--recursive']
```
Comment (thiagowfx) [inline .pre-commit-config.yaml:46]: If I remember correctly, `--recursive` isn't needed here.

`pre-commit` applies changes to all files included in the commit (`git diff` / `git show --stat`).

Comment (thiagowfx) [reply .pre-commit-config.yaml:46]: Just do `pre-commit run --all-files` once locally to apply it to all files in the repository, that should be enough.

File: .style.yapf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[style]
+based_on_style = pep8
+INDENT_DICTIONARY_VALUE=True
+ALLOW_SPLIT_BEFORE_DICT_VALUE=False
```
Comment (thiagowfx) [inline .style.yapf:4]:
```suggestion
based_on_style = pep8
indent_dictionary_value = True
allow_split_before_dict_value = False
```

(consistent case with `based_on_style`)

File: .pre-commit-config.yaml (hunk 1 above)
Comment (sadym-chromium) [reply .pre-commit-config.yaml]: done

**Review by thiagowfx — APPROVED** [review]

**Pull request merged**